discovery_engine_core/web-api/.env

```diff
@@ -1,3 +1,4 @@
 DE_SMBERT_VOCAB="assets/vocab.txt"
 DE_SMBERT_MODEL="assets/model.onnx"
 DE_DATA_PATH="assets/data.json"
+DE_POSTGRES_URL="postgresql://localhost?dbname=user&user=user&password=pw"
```

Comment:
❓ i have no idea how postgres is set up, so this might be a dumb question. is it on purpose that the `dbname` and `user` keys are set to the same `user` value? is the `pw` value for the `password` key some kind of dummy value for now or are all the values replaced during runtime or sth else?

Reply:
The value of `dbname` is just a placeholder but we can actually change it to `web_api`. The `user` and `password` values ​​come from the docker-compose file https://github.com/xaynetwork/xayn_discovery_engine/blob/a4ae96b4ea67569bc6e0df9645e6e86d531c3543/discovery_engine_core/web-api/docker-compose.yml#L7-L8
When team silver deploys the docker image, they can use the `DE_POSTGRES_URL` env variable to override it with a url that contains the prod credentials.

Reply:
thanks for the explanation! 👍